controllers/ExhibitsController.php

```diff
@@ -162,6 +162,22 @@ public function summaryAction()
         }
 
         fire_plugin_hook('show_exhibit', array('exhibit' => $exhibit));
+
+        if (!$exhibit->use_summary_page) {
+            $firstPage = $exhibit->getFirstTopPage();
+            if (null !== $firstPage) {
+                $this->_helper->redirector->gotoRoute(
+                    array(
+                        'controller' => 'exhibits',
+                        'action' => 'show',
```

Comment:
The controller and action are implied by the fact that you're using the "exhibitShow" route, so these two lines aren't needed.